package.json

```diff
@@ -162,5 +162,8 @@
       "build-js"
     ]
   },
-  "snyk": true
+  "snyk": true,
+  "dependencies": {
+    "uswds": "github:uswds/uswds#jm-readme-updates"
```

Comment:
let's use `github:uswds/uswds#develop` now

Reply:
Added in [cbf078c](https://github.com/uswds/uswds-site/pull/1582/commits/cbf078c459531ad8ffa8ad2291fec9e9d43e2aec)!